ui/desktop/app/routes/scopes/scope/projects/sessions.js

```diff
@@ -84,6 +85,28 @@ export default class ScopesScopeProjectsSessionsRoute extends Route {
 
   // =actions
 
+  /**
+   * A hack to force a refresh when coming from a child route back to the list route.
+   * This is needed because these model hooks are on the `sessions` route which is a
+   * parent to the `sessions.session` route so we never leave the route which means
+   * the model hooks won't rerun when going back to the list route.
+   *
+   * This could be alternatively solved by moving all the model fetching to
```

Comment:
out of curiosity- why didn't we implement the alternate solution?

Reply:
I tried it but ran into errors, it might be worth looking into refactoring it later

Reply:
I actually re-did it and it works now, I might have had an unrelated error from permissions testing, do we have a preference?

Reply:
I think I will actually move it out into the index even with this weird extra error because otherwise we force a user to have list permissions as well when trying to connect with just `read,cancel` permissions for sessions. If it's in the index, the list route won't get called during connect as it won't be in the direct hierarchy

Reply:
I figured out the bug and was able to move the model hook over, take another look.